src/openloco/station.h

```diff
@@ -76,25 +89,24 @@ namespace openloco
         uint16_t flags;
         town_id_t town;                                   // 0x2C
         station_cargo_stats cargo_stats[max_cargo_stats]; // 0x2E
-        uint16_t var_1CE;
-        uint16_t var_1D0;
-        uint16_t var_1D2;
-        uint16_t var_1D4;
-        uint8_t pad_1D6[0x3B0 - 0x1D6];
+        uint16_t stationTileSize;                         // 0x1CE
+        map_pos3 stationTiles[80];                        // 0x1D0
         uint8_t var_3B0;
         uint8_t var_3B1;
         uint8_t pad_3B2[0x3D2 - 0x3B2];
 
         bool empty() const { return name == string_ids::null; }
         station_id_t id() const;
         void update();
-        uint32_t calc_accepted_cargo(uint16_t ax = 0xFFFF);
+        uint32_t calcAcceptedCargo(CargoSearchState& cargoSearchState);
+        uint32_t calcAcceptedCargo(CargoSearchState& cargoSearchState, map_pos location, uint32_t ebx);
         void sub_48F7D1();
         void getStatusString(const char* buffer);
         bool update_cargo();
         int32_t calculate_cargo_rating(const station_cargo_stats& cargo) const;
         void invalidate();
         void invalidate_window();
+        void setStationCatchmentDisplay(uint8_t flags);
```

Comment:
As this is part of the station struct anyway, I think we might as well leave out 'Station' from the name:
```suggestion
        void setCatchmentDisplay(uint8_t flags);
```